example/screens/UsageDisabled.tsx

```diff
@@ -8,6 +8,7 @@ const DisableableHeaderButton = (props) => (
     {...props}
     onPress={props.onPress}
     disabled={props.disabled}
+    enabled={!props.disabled}
```

Comment:
we should not be passing both disabled and enabled prop to a component. Please just pass disabled and use that in the Touchable, thanks!